src/app/config/selectors.js

```diff
@@ -7,7 +7,7 @@ export const getCollections = state => state.collections.all;
 export const getSearch = state => state.search;
 
 export const getMappedCollections = createSelector(getCollections, collections => {
-    if (!collections) return [];
+    if (!collections || collections.length === 0 || typeof collections === "string") return [];
```

Comment:
this is temporary as collection api returns string with error message you are not authenticated. If this is removed the app crashes